src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1493,7 +1493,6 @@ public final Response getEventOverviews(@Context final HttpServletRequest reques
         if (filterOption.equals(EventFilterOption.FUTURE)) {
           DateRangeFilterInstruction anyEventsFromNow = new DateRangeFilterInstruction(Instant.now(), null);
           filterInstructions.put(ENDDATE_FIELDNAME, anyEventsFromNow);
-          sortInstructions.put(DATE_FIELDNAME, SortOrder.ASC);
```

Comment:
Just a quick note - there appears to be a descending sort already applied at the start of the function (ln 1479), which the removed one overwrote. Do you just want to maintain the descending sort or eliminate it entirely?

Reply:
Yeah that's intentional as it was requested to change the display order; with that line it was sorting "future" events differently which didn't work well once pagination was implemented on front end.